node and nodepurejs interop failing to build

```
539d7b65b8d5: Pull complete
d3f935c306e4: Pull complete
4ec1e1aced13: Pull complete
b1a388be0713: Pull complete
6768c844affa: Pull complete
5210913f321b: Pull complete
07ede02d8973: Pull complete
b89610bcf4f6: Pull complete
22b19ebf7616: Pull complete
aa219cfd37c0: Pull complete
31b1b64645af: Pull complete
Digest: sha256:2c34cfefc509ab1b4abde185afcb84f65eb4be51badf8f10ba7abf385863fe2f
Status: Downloaded newer image for grpctesting/grpc_interop_node_29f61ea6f66bf0c5c32d1b672a53de6aef7dbd90:latest
real	0m29.578s
user	0m0.148s
sys	0m0.051s
+ git branch -f jenkins-docker
++ uuidgen
+ CONTAINER_NAME=build_grpc_interop_node_02e8dbd7-48f2-44b1-8b25-6ecc5a4de336
+ docker run -e CCACHE_DIR=/tmp/ccache -e 'THIS_IS_REALLY_NEEDED=see https://github.com/docker/docker/issues/14203 for why docker is awful' -e 'THIS_IS_REALLY_NEEDED_ONCE_AGAIN=For issue 4835. See https://github.com/docker/docker/issues/14203 for why docker is awful' -i -v /tmpfs/src/github/grpc:/var/local/jenkins/grpc:ro -v /tmpfs/src/github/grpc-java:/var/local/jenkins/grpc-java:ro -v /tmpfs/src/github/grpc-go:/var/local/jenkins/grpc-go:ro -v /tmpfs/src/github/grpc-dart:/var/local/jenkins/grpc-dart:ro -v /tmpfs/src/github/grpc-node:/var/local/jenkins/grpc-node:ro -v /home/kbuilder/service_account:/var/local/jenkins/service_account:ro -v /tmp/ccache:/tmp/ccache --name=build_grpc_interop_node_02e8dbd7-48f2-44b1-8b25-6ecc5a4de336 grpctesting/grpc_interop_node_29f61ea6f66bf0c5c32d1b672a53de6aef7dbd90 bash -l /var/local/jenkins/grpc/tools/dockerfile/interoptest/grpc_interop_node/build_interop.sh
stdin: is not a tty
Cloning into '/var/local/git/grpc-node'...
done.
Entering 'packages/grpc-native-core/deps/grpc'
Submodule 'packages/grpc-native-core/deps/grpc' (https://github.com/grpc/grpc.git) registered for path 'packages/grpc-native-core/deps/grpc'
Cloning into 'packages/grpc-native-core/deps/grpc'...
Submodule path 'packages/grpc-native-core/deps/grpc': checked out 'aef957950aa3390e3516531ca2d783de8fc9333b'
Submodule 'third_party/abseil-cpp' (https://github.com/abseil/abseil-cpp) registered for path 'third_party/abseil-cpp'
Submodule 'third_party/benchmark' (https://github.com/google/benchmark) registered for path 'third_party/benchmark'
Submodule 'third_party/bloaty' (https://github.com/google/bloaty.git) registered for path 'third_party/bloaty'
Submodule 'third_party/boringssl' (https://github.com/google/boringssl.git) registered for path 'third_party/boringssl'
Submodule 'third_party/boringssl-with-bazel' (https://github.com/google/boringssl.git) registered for path 'third_party/boringssl-with-bazel'
Submodule 'third_party/cares/cares' (https://github.com/c-ares/c-ares.git) registered for path 'third_party/cares/cares'
Submodule 'third_party/gflags' (https://github.com/gflags/gflags.git) registered for path 'third_party/gflags'
Submodule 'third_party/googletest' (https://github.com/google/googletest.git) registered for path 'third_party/googletest'
Submodule 'third_party/protobuf' (https://github.com/google/protobuf.git) registered for path 'third_party/protobuf'
Submodule 'third_party/zlib' (https://github.com/madler/zlib) registered for path 'third_party/zlib'
Cloning into 'third_party/abseil-cpp'...
warning: no common commits
Submodule path 'packages/grpc-native-core/deps/grpc/third_party/abseil-cpp': checked out 'cc4bed2d74f7c8717e31f9579214ab52a9c9c610'
Cloning into 'third_party/benchmark'...
warning: no common commits
Submodule path 'packages/grpc-native-core/deps/grpc/third_party/benchmark': checked out '5b7683f49e1e9223cf9927b24f6fd3d6bd82e3f8'
Cloning into 'third_party/bloaty'...
warning: no common commits
Submodule path 'packages/grpc-native-core/deps/grpc/third_party/bloaty': checked out '73594cde8c9a52a102c4341c244c833aa61b9c06'
Submodule 'third_party/googletest' (https://github.com/google/googletest.git) registered for path 'third_party/googletest'
Submodule 'third_party/libFuzzer' (https://chromium.googlesource.com/chromium/llvm-project/llvm/lib/Fuzzer) registered for path 'third_party/libFuzzer'
Submodule 'third_party/re2' (https://github.com/google/re2) registered for path 'third_party/re2'
Cloning into 'third_party/googletest'...
warning: no common commits
Submodule path 'packages/grpc-native-core/deps/grpc/third_party/bloaty/third_party/googletest': checked out 'a2b8a8e07628e5fd60644b6dd99c1b5e7d7f1f47'
Cloning into 'third_party/libFuzzer'...
warning: no common commits
Submodule path 'packages/grpc-native-core/deps/grpc/third_party/bloaty/third_party/libFuzzer': checked out '1b543d6e5073b56be214394890c9193979a3d7e1'
Cloning into 'third_party/re2'...
warning: no common commits
Submodule path 'packages/grpc-native-core/deps/grpc/third_party/bloaty/third_party/re2': checked out 'c964d9b07816f9b52ca692e23b8b3dba33beec54'
Cloning into 'third_party/boringssl'...
warning: no common commits
Submodule path 'packages/grpc-native-core/deps/grpc/third_party/boringssl': checked out 'a20bb7ff8bb5057065a2e7941249773f9676cf45'
Cloning into 'third_party/boringssl-with-bazel'...
warning: no common commits
Submodule path 'packages/grpc-native-core/deps/grpc/third_party/boringssl-with-bazel': checked out '886e7d75368e3f4fab3f4d0d3584e4abfc557755'
Cloning into 'third_party/cares/cares'...
warning: no common commits
Submodule path 'packages/grpc-native-core/deps/grpc/third_party/cares/cares': checked out '3be1924221e1326df520f8498d704a5c4c8d0cce'
Cloning into 'third_party/gflags'...
warning: no common commits
Submodule path 'packages/grpc-native-core/deps/grpc/third_party/gflags': checked out '30dbc81fb5ffdc98ea9b14b1918bfe4e8779b26e'
Submodule 'doc' (https://github.com/gflags/gflags.git) registered for path 'doc'
Cloning into 'doc'...
warning: no common commits
Submodule path 'packages/grpc-native-core/deps/grpc/third_party/gflags/doc': checked out '971dd2a4fadac9cdab174c523c22df79efd63aa5'
Cloning into 'third_party/googletest'...
warning: no common commits
Submodule path 'packages/grpc-native-core/deps/grpc/third_party/googletest': checked out 'ec44c6c1675c25b9827aacd08c02433cccde7780'
Cloning into 'third_party/protobuf'...
warning: no common commits
Submodule path 'packages/grpc-native-core/deps/grpc/third_party/protobuf': checked out 'b5fbb742af122b565925987e65c08957739976a7'
Submodule 'third_party/benchmark' (https://github.com/google/benchmark.git) registered for path 'third_party/benchmark'
Cloning into 'third_party/benchmark'...
warning: no common commits
Submodule path 'packages/grpc-native-core/deps/grpc/third_party/protobuf/third_party/benchmark': checked out '360e66c1c4777c99402cf8cd535aa510fee16573'
Cloning into 'third_party/zlib'...
warning: no common commits
Submodule path 'packages/grpc-native-core/deps/grpc/third_party/zlib': checked out 'cacf7f1d4e3d44d871b605da3b647f07d718623f'
npm WARN deprecated gulp-util@3.0.8: gulp-util is deprecated - replace it, following the guidelines at https://medium.com/gulpjs/gulp-util-ca3b1f9f9ac5
npm WARN deprecated graceful-fs@3.0.11: please upgrade to graceful-fs 4 for compatibility with current and future versions of Node.js
npm WARN deprecated minimatch@2.0.10: Please update to minimatch 3.0.2 or higher to avoid a RegExp DoS issue
npm WARN deprecated minimatch@0.2.14: Please update to minimatch 3.0.2 or higher to avoid a RegExp DoS issue
npm WARN deprecated graceful-fs@1.2.3: please upgrade to graceful-fs 4 for compatibility with current and future versions of Node.js
npm WARN deprecated natives@1.1.3: This module relies on Node.js's internals and will break at some point. Do not use it, and update to graceful-fs@4.x.
npm WARN notice [SECURITY] minimatch has 1 high vulnerability. Go here for more details: https://nodesecurity.io/advisories?search=minimatch&version=0.2.14 - Run `npm i npm@latest -g` to upgrade your npm version, and then `npm audit` to get more info.
npm WARN notice [SECURITY] minimatch has 1 high vulnerability. Go here for more details: https://nodesecurity.io/advisories?search=minimatch&version=2.0.10 - Run `npm i npm@latest -g` to upgrade your npm version, and then `npm audit` to get more info.
npm WARN notice [SECURITY] lodash has 1 low vulnerability. Go here for more details: https://nodesecurity.io/advisories?search=lodash&version=1.0.2 - Run `npm i npm@latest -g` to upgrade your npm version, and then `npm audit` to get more info.
/root/.nvm/versions/node/v9.11.1/bin/gulp -> /root/.nvm/versions/node/v9.11.1/lib/node_modules/gulp/bin/gulp.js
/root/.nvm/versions/node/v9.11.1/bin/node-gyp -> /root/.nvm/versions/node/v9.11.1/lib/node_modules/node-gyp/bin/node-gyp.js
+ gulp@3.9.1
+ node-gyp@3.6.2
added 355 packages from 217 contributors in 9.725s
npm WARN deprecated gulp-util@3.0.8: gulp-util is deprecated - replace it, following the guidelines at https://medium.com/gulpjs/gulp-util-ca3b1f9f9ac5
npm WARN deprecated graceful-fs@3.0.11: please upgrade to graceful-fs 4 for compatibility with current and future versions of Node.js
npm WARN deprecated minimatch@2.0.10: Please update to minimatch 3.0.2 or higher to avoid a RegExp DoS issue
npm WARN deprecated natives@1.1.3: This module relies on Node.js's internals and will break at some point. Do not use it, and update to graceful-fs@4.x.
npm WARN deprecated minimatch@0.2.14: Please update to minimatch 3.0.2 or higher to avoid a RegExp DoS issue
npm WARN deprecated graceful-fs@1.2.3: please upgrade to graceful-fs 4 for compatibility with current and future versions of Node.js
npm WARN notice [SECURITY] debug has 1 low vulnerability. Go here for more details: https://nodesecurity.io/advisories?search=debug&version=2.6.8 - Run `npm i npm@latest -g` to upgrade your npm version, and then `npm audit` to get more info.
npm WARN notice [SECURITY] lodash has 1 low vulnerability. Go here for more details: https://nodesecurity.io/advisories?search=lodash&version=3.7.0 - Run `npm i npm@latest -g` to upgrade your npm version, and then `npm audit` to get more info.
npm WARN notice [SECURITY] growl has 1 critical vulnerability. Go here for more details: https://nodesecurity.io/advisories?search=growl&version=1.9.2 - Run `npm i npm@latest -g` to upgrade your npm version, and then `npm audit` to get more info.
npm ERR! code E503
npm ERR! 503 Connection reset by peer: @gulp-sourcemaps/map-sources@1.X
npm ERR! A complete log of this run can be found in:
npm ERR!     /tmp/npm-cache/_logs/2018-05-18T17_58_55_595Z-debug.log
+ EXITCODE=1
+ docker rm -f build_grpc_interop_node_02e8dbd7-48f2-44b1-8b25-6ecc5a4de336
build_grpc_interop_node_02e8dbd7-48f2-44b1-8b25-6ecc5a4de336
+ exit 1
2018-05-18 10:58:59,477 FAILED: build_docker_node [ret=1, pid=6318, time=652.3sec]
```

```
fdedbd005ebe: Pull complete
a1d05393a7ec: Pull complete
7d3b53d4a7e7: Pull complete
414b8d0be098: Pull complete
Digest: sha256:f4eeaf58f01c5c0af4b347bec8d3f06359abad9cdcf011d3d499fb87c81566b4
Status: Downloaded newer image for grpctesting/grpc_interop_nodepurejs_0ac9adebbbaa9dcad7f175b96cd5aca9902fc686:latest
real	3m46.350s
user	0m0.152s
sys	0m0.095s
+ git branch -f jenkins-docker
++ uuidgen
+ CONTAINER_NAME=build_grpc_interop_nodepurejs_a5bd7a72-a9e5-41b0-b918-36d54be3d526
+ docker run -e CCACHE_DIR=/tmp/ccache -e 'THIS_IS_REALLY_NEEDED=see https://github.com/docker/docker/issues/14203 for why docker is awful' -e 'THIS_IS_REALLY_NEEDED_ONCE_AGAIN=For issue 4835. See https://github.com/docker/docker/issues/14203 for why docker is awful' -i -v /tmpfs/src/github/grpc:/var/local/jenkins/grpc:ro -v /tmpfs/src/github/grpc-java:/var/local/jenkins/grpc-java:ro -v /tmpfs/src/github/grpc-go:/var/local/jenkins/grpc-go:ro -v /tmpfs/src/github/grpc-dart:/var/local/jenkins/grpc-dart:ro -v /tmpfs/src/github/grpc-node:/var/local/jenkins/grpc-node:ro -v /home/kbuilder/service_account:/var/local/jenkins/service_account:ro -v /tmp/ccache:/tmp/ccache --name=build_grpc_interop_nodepurejs_a5bd7a72-a9e5-41b0-b918-36d54be3d526 grpctesting/grpc_interop_nodepurejs_0ac9adebbbaa9dcad7f175b96cd5aca9902fc686 bash -l /var/local/jenkins/grpc/tools/dockerfile/interoptest/grpc_interop_nodepurejs/build_interop.sh
stdin: is not a tty
Cloning into '/var/local/git/grpc-node'...
done.
Entering 'packages/grpc-native-core/deps/grpc'
Submodule 'packages/grpc-native-core/deps/grpc' (https://github.com/grpc/grpc.git) registered for path 'packages/grpc-native-core/deps/grpc'
Cloning into 'packages/grpc-native-core/deps/grpc'...
Submodule path 'packages/grpc-native-core/deps/grpc': checked out 'aef957950aa3390e3516531ca2d783de8fc9333b'
Submodule 'third_party/abseil-cpp' (https://github.com/abseil/abseil-cpp) registered for path 'third_party/abseil-cpp'
Submodule 'third_party/benchmark' (https://github.com/google/benchmark) registered for path 'third_party/benchmark'
Submodule 'third_party/bloaty' (https://github.com/google/bloaty.git) registered for path 'third_party/bloaty'
Submodule 'third_party/boringssl' (https://github.com/google/boringssl.git) registered for path 'third_party/boringssl'
Submodule 'third_party/boringssl-with-bazel' (https://github.com/google/boringssl.git) registered for path 'third_party/boringssl-with-bazel'
Submodule 'third_party/cares/cares' (https://github.com/c-ares/c-ares.git) registered for path 'third_party/cares/cares'
Submodule 'third_party/gflags' (https://github.com/gflags/gflags.git) registered for path 'third_party/gflags'
Submodule 'third_party/googletest' (https://github.com/google/googletest.git) registered for path 'third_party/googletest'
Submodule 'third_party/protobuf' (https://github.com/google/protobuf.git) registered for path 'third_party/protobuf'
Submodule 'third_party/zlib' (https://github.com/madler/zlib) registered for path 'third_party/zlib'
Cloning into 'third_party/abseil-cpp'...
warning: no common commits
Submodule path 'packages/grpc-native-core/deps/grpc/third_party/abseil-cpp': checked out 'cc4bed2d74f7c8717e31f9579214ab52a9c9c610'
Cloning into 'third_party/benchmark'...
warning: no common commits
Submodule path 'packages/grpc-native-core/deps/grpc/third_party/benchmark': checked out '5b7683f49e1e9223cf9927b24f6fd3d6bd82e3f8'
Cloning into 'third_party/bloaty'...
warning: no common commits
Submodule path 'packages/grpc-native-core/deps/grpc/third_party/bloaty': checked out '73594cde8c9a52a102c4341c244c833aa61b9c06'
Submodule 'third_party/googletest' (https://github.com/google/googletest.git) registered for path 'third_party/googletest'
Submodule 'third_party/libFuzzer' (https://chromium.googlesource.com/chromium/llvm-project/llvm/lib/Fuzzer) registered for path 'third_party/libFuzzer'
Submodule 'third_party/re2' (https://github.com/google/re2) registered for path 'third_party/re2'
Cloning into 'third_party/googletest'...
warning: no common commits
Submodule path 'packages/grpc-native-core/deps/grpc/third_party/bloaty/third_party/googletest': checked out 'a2b8a8e07628e5fd60644b6dd99c1b5e7d7f1f47'
Cloning into 'third_party/libFuzzer'...
warning: no common commits
Submodule path 'packages/grpc-native-core/deps/grpc/third_party/bloaty/third_party/libFuzzer': checked out '1b543d6e5073b56be214394890c9193979a3d7e1'
Cloning into 'third_party/re2'...
warning: no common commits
Submodule path 'packages/grpc-native-core/deps/grpc/third_party/bloaty/third_party/re2': checked out 'c964d9b07816f9b52ca692e23b8b3dba33beec54'
Cloning into 'third_party/boringssl'...
warning: no common commits
Submodule path 'packages/grpc-native-core/deps/grpc/third_party/boringssl': checked out 'a20bb7ff8bb5057065a2e7941249773f9676cf45'
Cloning into 'third_party/boringssl-with-bazel'...
warning: no common commits
Submodule path 'packages/grpc-native-core/deps/grpc/third_party/boringssl-with-bazel': checked out '886e7d75368e3f4fab3f4d0d3584e4abfc557755'
Cloning into 'third_party/cares/cares'...
warning: no common commits
Submodule path 'packages/grpc-native-core/deps/grpc/third_party/cares/cares': checked out '3be1924221e1326df520f8498d704a5c4c8d0cce'
Cloning into 'third_party/gflags'...
warning: no common commits
Submodule path 'packages/grpc-native-core/deps/grpc/third_party/gflags': checked out '30dbc81fb5ffdc98ea9b14b1918bfe4e8779b26e'
Submodule 'doc' (https://github.com/gflags/gflags.git) registered for path 'doc'
Cloning into 'doc'...
warning: no common commits
Submodule path 'packages/grpc-native-core/deps/grpc/third_party/gflags/doc': checked out '971dd2a4fadac9cdab174c523c22df79efd63aa5'
Cloning into 'third_party/googletest'...
warning: no common commits
Submodule path 'packages/grpc-native-core/deps/grpc/third_party/googletest': checked out 'ec44c6c1675c25b9827aacd08c02433cccde7780'
Cloning into 'third_party/protobuf'...
warning: no common commits
Submodule path 'packages/grpc-native-core/deps/grpc/third_party/protobuf': checked out 'b5fbb742af122b565925987e65c08957739976a7'
Submodule 'third_party/benchmark' (https://github.com/google/benchmark.git) registered for path 'third_party/benchmark'
Cloning into 'third_party/benchmark'...
warning: no common commits
Submodule path 'packages/grpc-native-core/deps/grpc/third_party/protobuf/third_party/benchmark': checked out '360e66c1c4777c99402cf8cd535aa510fee16573'
Cloning into 'third_party/zlib'...
warning: no common commits
Submodule path 'packages/grpc-native-core/deps/grpc/third_party/zlib': checked out 'cacf7f1d4e3d44d871b605da3b647f07d718623f'
npm WARN deprecated gulp-util@3.0.8: gulp-util is deprecated - replace it, following the guidelines at https://medium.com/gulpjs/gulp-util-ca3b1f9f9ac5
npm WARN deprecated graceful-fs@3.0.11: please upgrade to graceful-fs 4 for compatibility with current and future versions of Node.js
npm WARN deprecated minimatch@2.0.10: Please update to minimatch 3.0.2 or higher to avoid a RegExp DoS issue
npm WARN deprecated minimatch@0.2.14: Please update to minimatch 3.0.2 or higher to avoid a RegExp DoS issue
npm WARN deprecated graceful-fs@1.2.3: please upgrade to graceful-fs 4 for compatibility with current and future versions of Node.js
npm WARN deprecated natives@1.1.3: This module relies on Node.js's internals and will break at some point. Do not use it, and update to graceful-fs@4.x.
npm WARN notice [SECURITY] minimatch has 1 high vulnerability. Go here for more details: https://nodesecurity.io/advisories?search=minimatch&version=0.2.14 - Run `npm i npm@latest -g` to upgrade your npm version, and then `npm audit` to get more info.
npm WARN notice [SECURITY] minimatch has 1 high vulnerability. Go here for more details: https://nodesecurity.io/advisories?search=minimatch&version=2.0.10 - Run `npm i npm@latest -g` to upgrade your npm version, and then `npm audit` to get more info.
npm WARN notice [SECURITY] lodash has 1 low vulnerability. Go here for more details: https://nodesecurity.io/advisories?search=lodash&version=1.0.2 - Run `npm i npm@latest -g` to upgrade your npm version, and then `npm audit` to get more info.
/root/.nvm/versions/node/v9.8.0/bin/gulp -> /root/.nvm/versions/node/v9.8.0/lib/node_modules/gulp/bin/gulp.js
+ gulp@3.9.1
added 255 packages from 164 contributors in 9.158s
npm WARN deprecated gulp-util@3.0.8: gulp-util is deprecated - replace it, following the guidelines at https://medium.com/gulpjs/gulp-util-ca3b1f9f9ac5
npm WARN deprecated graceful-fs@3.0.11: please upgrade to graceful-fs 4 for compatibility with current and future versions of Node.js
npm WARN deprecated minimatch@2.0.10: Please update to minimatch 3.0.2 or higher to avoid a RegExp DoS issue
npm WARN deprecated natives@1.1.3: This module relies on Node.js's internals and will break at some point. Do not use it, and update to graceful-fs@4.x.
npm WARN deprecated minimatch@0.2.14: Please update to minimatch 3.0.2 or higher to avoid a RegExp DoS issue
npm WARN deprecated graceful-fs@1.2.3: please upgrade to graceful-fs 4 for compatibility with current and future versions of Node.js
npm WARN notice [SECURITY] debug has 1 low vulnerability. Go here for more details: https://nodesecurity.io/advisories?search=debug&version=2.6.8 - Run `npm i npm@latest -g` to upgrade your npm version, and then `npm audit` to get more info.
npm WARN notice [SECURITY] lodash has 1 low vulnerability. Go here for more details: https://nodesecurity.io/advisories?search=lodash&version=3.7.0 - Run `npm i npm@latest -g` to upgrade your npm version, and then `npm audit` to get more info.
npm WARN notice [SECURITY] growl has 1 critical vulnerability. Go here for more details: https://nodesecurity.io/advisories?search=growl&version=1.9.2 - Run `npm i npm@latest -g` to upgrade your npm version, and then `npm audit` to get more info.
npm notice created a lockfile as package-lock.json. You should commit this file.
added 954 packages from 822 contributors in 282.805s
[17:54:55] Requiring external module ts-node/register
[17:54:56] Using gulpfile /var/local/git/grpc-node/gulpfile.ts
[17:54:56] Starting 'js.core.install'...
> @grpc/grpc-js@0.1.0 prepare /var/local/git/grpc-node/packages/grpc-js-core
> npm run compile
> @grpc/grpc-js@0.1.0 compile /var/local/git/grpc-node/packages/grpc-js-core
> tsc -p .
npm notice created a lockfile as package-lock.json. You should commit this file.
npm WARN @grpc/grpc-js@0.1.0 No repository field.
added 179 packages from 110 contributors in 51.257s
[17:55:48] Finished 'js.core.install' after 52 s
[17:55:49] Requiring external module ts-node/register
[17:55:50] Using gulpfile /var/local/git/grpc-node/gulpfile.ts
[17:55:50] Starting 'protobuf.install'...
npm ERR! code E500
npm ERR! 500 Internal Server Error: @protobufjs/inquire@^1.1.0
npm ERR! A complete log of this run can be found in:
npm ERR!     /tmp/npm-cache/_logs/2018-05-18T18_04_58_604Z-debug.log
[18:04:58] 'protobuf.install' errored after 9.13 min
[18:04:58] Error: Command failed: npm install --unsafe-perm
    at Promise.all.then.arr (/var/local/git/grpc-node/node_modules/execa/index.js:236:11)
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:118:7)
+ EXITCODE=1
+ docker rm -f build_grpc_interop_nodepurejs_a5bd7a72-a9e5-41b0-b918-36d54be3d526
build_grpc_interop_nodepurejs_a5bd7a72-a9e5-41b0-b918-36d54be3d526
+ exit 1
2018-05-18 11:05:00,352 FAILED: build_docker_nodepurejs [ret=1, pid=8849, time=1264.2sec]
```

https://source.cloud.google.com/results/invocations/b066f797-410e-47c0-a120-b7bd3c2d4164/targets/grpc%2Fcore%2Fpull_request%2Flinux%2Fgrpc_interop_tocloud/log

--- a/tools/dockerfile/interoptest/grpc_interop_node/build_interop.sh
+++ b/tools/dockerfile/interoptest/grpc_interop_node/build_interop.sh
@@ -29,6 +29,4 @@ cp -r /var/local/jenkins/service_account $HOME || true
 cd /var/local/git/grpc-node
 
 # build Node interop client & server
-npm install -g node-gyp gulp
-npm install
-gulp setup
+./setup_interop.sh

--- a/tools/dockerfile/interoptest/grpc_interop_nodepurejs/build_interop.sh
+++ b/tools/dockerfile/interoptest/grpc_interop_nodepurejs/build_interop.sh
@@ -29,8 +29,4 @@ cp -r /var/local/jenkins/service_account $HOME || true
 cd /var/local/git/grpc-node
 
 # build Node interop client & server
-npm install -g gulp
-npm install
-gulp js.core.install
-gulp protobuf.install
-gulp internal.test.install
+./setup_interop_purejs.sh

